classes/Http/Controller/ForgotController.php

```diff
@@ -224,24 +202,18 @@ public function updatePasswordAction(Request $request)
         }
 
         try {
-            /** @var AccountManagement $accounts */
-            $accounts = $this->service(AccountManagement::class);
-
-            $user = $accounts->findById($userId);
+            $user = $this->accounts->findById($userId);
         } catch (\RuntimeException $e) {
             echo $e;
             die();
         }
 
-        /** @var Session\Session $session */
-        $session = $this->service('session');
```

Comment:
Since we would otherwise be invoking `$request->getSession()` multiple times, can we maybe just use `$session` here and assign it `$request->getSession()`?

Reply:
See the discussion above.